types/pdfmake/index.d.ts

```diff
@@ -23,56 +23,55 @@ declare module "pdfmake/build/pdfmake" {
         vfs?: any
     ): TCreatedPdf;
 
-    enum PageSize {
-        A0_x_4 = "4A0",
-        A0_x_2 = "2A0",
-        AO = "A0",
-        A1 = "A1",
-        A2 = "A2",
-        A3 = "A3",
-        A4 = "A4",
-        A5 = "A5",
-        A6 = "A6",
-        A7 = "A7",
-        A8 = "A8",
-        A9 = "A9",
-        A1O = "A10",
-        BO = "B0",
-        B1 = "B1",
-        B2 = "B2",
-        B3 = "B3",
-        B4 = "B4",
-        B5 = "B5",
-        B6 = "B6",
-        B7 = "B7",
-        B8 = "B8",
-        B9 = "B9",
-        B1O = "B10",
-        CO = "C0",
-        C1 = "C1",
-        C2 = "C2",
-        C3 = "C3",
-        C4 = "C4",
-        C5 = "C5",
-        C6 = "C6",
-        C7 = "C7",
-        C8 = "C8",
-        C9 = "C9",
-        C1O = "C10",
-        RA1 = "RA1",
-        RA2 = "RA2",
-        RA3 = "RA3",
-        RA4 = "RA4",
-        SRA1 = "SRA1",
-        SRA2 = "SRA2",
-        SRA3 = "SRA3",
-        SRA4 = "SRA4",
-        EXECUTIVE = "EXECUTIVE",
-        FOLIO = "FOLIO",
-        LEGAL = "LEGAL",
-        LETTER = "LETTER",
-        TABLOID = "TABLOID"
-    }
+    type PageSize =
+        | '4A0'
+        | '2A0'
+        | 'A0'
+        | 'A1'
+        | 'A2'
+        | 'A3'
+        | 'A4'
+        | 'A5'
+        | 'A6'
+        | 'A7'
+        | 'A8'
+        | 'A9'
+        | 'A10'
+        | 'B0'
+        | 'B1'
+        | 'B2'
+        | 'B3'
+        | 'B4'
+        | 'B5'
+        | 'B6'
+        | 'B7'
+        | 'B8'
+        | 'B9'
+        | 'B10'
+        | 'C0'
+        | 'C1'
+        | 'C2'
+        | 'C3'
+        | 'C4'
+        | 'C5'
+        | 'C6'
+        | 'C7'
+        | 'C8'
+        | 'C9'
+        | 'C10'
+        | 'RA1'
+        | 'RA2'
+        | 'RA3'
+        | 'RA4'
+        | 'SRA1
```

Comment:
You missed a `'` here

Reply:
The reason is that the string values are not accessible via `pdfmake.PageSize`. So using `pdfmake.PageSize.A4` resulted in valid linting but compile errors. Using a string instead throws linting errors but does compile.